skdate/ts/src/sk_date.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  int,
+  ptr,
+  Environment,
+  Links,
+  ToWasmManager,
+  Utils,
+  FileSystem,
+  Options,
+} from "#std/sk_types";
+
+interface ToWasm {
+  SKIP_localetimezone: (year: int, month: int, day: int) => int;
+  SKIP_localetimezonename: (year: int, month: int, day: int) => ptr;
+  SKIP_locale: (code: int, value: int) => ptr;
+  SKIP_JS_currenttimemillis: () => number;
+}
+
+class LinksImpl implements Links {
+  SKIP_localetimezone: (year: int, month: int, day: int) => int;
+  SKIP_localetimezonename: (year: int, month: int, day: int) => ptr;
+  SKIP_locale: (code: int, value: int) => ptr;
+
+  constructor() {}
+
+  complete = (utils: Utils, exports: object) => {
+    this.SKIP_localetimezone = (year: int, month: int, day: int) => {
+      let date = new Date(year, month - 1, day);
+      let offset = -(date.getTimezoneOffset() * 60);
+      return offset;
+    };
+    this.SKIP_localetimezonename = (year: int, month: int, day: int) => {
+      let date = new Date(year, month - 1, day);
+      const timeZone = date.toLocaleDateString(undefined, {
+        timeZoneName: "short",
+      });
+      return utils.exportString(timeZone);
+    };
+    this.SKIP_locale = (code: int, value: int) => {
+      let str = String.fromCharCode(code);
+      if (str == "B" || str == "b") {
+        let date = new Date(Date.UTC(2023, 0 + (value - 1), 1)); // January
```

Comment:
I'm confused -- why this particular constant (and the others below)?

Reply:
Comment added